v3/docs/01-getting-started/a-overview/index.mdx

```diff
@@ -1,66 +1,61 @@
 ---
 id: 001
-title: Welcome
+
+title: Introduction to Substrate
 slug: /v3/getting-started/overview
 version: '3.0'
 section: docs
 category: getting started
 keywords: []
 ---
+# Overview
+
+This section provides an overview of Substrate – structure, working, and usage. 
```

Comment:
```suggestion
This section provides an overview of Substrate – structure, working, and usage. 
```
Generally, headings should always contain some real content. Here, you have a H1 top-level heading with a single introductory sentence. I think what you have as "What is Substrate?" could go here as a more useful introductory paragraph.